docs/API_ref.md

```diff
@@ -1,70 +1,77 @@
+
+| Property           | Type       | Description                                                                 |
+|--------------------|------------|-----------------------------------------------------------------------------|
+| enable             | *[bool](https://pkg.go.dev/builtin#bool)      | Enables backup/restore using Restic. If set to false, snapshots are needed. |
+| supplementalGroups | [][int64](https://pkg.go.dev/builtin#int64)    | SupplementalGroups defines the linux groups to be applied to the Restic Pod |
+| timeout            | [string](https://pkg.go.dev/builtin#string)     | Timeout defines the Restic timeout, default value is 1h                     |
+| PodConfig          | *[PodConfig](https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220722142130-a02487302a02/api/v1alpha1#PodConfig) | Restic Pod specific configuration                                           |
+
+### [PodConfig](https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220722142130-a02487302a02/api/v1alpha1#PodConfig)
 
 | Property            | Type                                                                                      | Description                                                                                                                               |
 |---------------------|-------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
-| nodeSelector        | map [ string ] string                                                                     | NodeSelector defines the nodeSelector to be supplied to Velero/Restic podSpec                                                             |
-| tolerations         | [[]corev1. Toleration](https://pkg.go.dev/k8s.io/api/core/v1#Toleration)                  | Tolerations defines the list of tolerations to be applied to Velero Deployment/Restic daemonset                                                             |
+| nodeSelector        | map [ [string](https://pkg.go.dev/builtin#string) ] [string](https://pkg.go.dev/builtin#string)                                                                     | NodeSelector defines the nodeSelector to be supplied to Velero/Restic podSpec                                                             |
+| tolerations         | [][corev1.Toleration](https://pkg.go.dev/k8s.io/api/core/v1#Toleration)                  | Tolerations defines the list of tolerations to be applied to Velero Deployment/Restic daemonset                                                             |
 | resourceAllocations | [corev1.ResourceRequirements](https://pkg.go.dev/k8s.io/api/core/v1#ResourceRequirements) | Set specific resource  `limits`  and  `requests`  for the Velero/Restic pods. For more information, go  [here] ( config/resource_req_limits.md). |
 
```

Comment:
Should this type include the parameter "labels"?

Reply:
Thanks!